<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBackupDialog</class>
 <widget class="QDialog" name="AddBackupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add backup folder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#macOSBigSurTitleLabel
{
  color: #242424;
  font-family: &quot;SF Pro Text&quot;;
  font-size: 14px;
}

#gbBackupLocation
{
  border: 1px solid rgba(0,0,0,0.1);
  border-radius: 8px;
  background-color: rgba(0,0,0,0.05);
}

#gbBackupLocation
{
 margin-right: 5px;
}

#folderLineEdit
{
padding-left:3px;
}

#formLabel_1, #formLabel_2
{
  color: #242424;
  font-family: &quot;SF Pro Text&quot;;
  font-size: 13px;
}

#backupToLabel
{
  color: #333333;
  font-family: &quot;Helvetica Neue&quot;;
  font-size: 12px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>28</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QLabel" name="embeddedTitleLabel">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#embeddedTitleLabel{
margin-left: 29px;
margin-right: 29px;
}</string>
     </property>
     <property name="text">
      <string>Add backup folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>32</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>32</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>20</number>
        </property>
        <item row="0" column="2">
         <widget class="QPushButton" name="changeButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Choose</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2" alignment="Qt::AlignTop">
         <widget class="QLabel" name="backupToLabel">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Backup to:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Backup folder:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="folderLineEdit">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>23</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Choose folder</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="styleSheet">
         <string notr="true">*{
color:#dadada;
margin-left: 32px;
margin-right: 32px;
}</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="bAddExclusions">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Add exclusions</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
